apps/frontend/src/modules/education/keyboards/components/KeyboardCard.css

```
@@ -0,0 +1,84 @@
+  animation: renderFlashInitial 0.6s ease-out;
+}
+
+/* RE-renders - gren flash */
+.keyboard-card.render-update {
+  animation: renderFlashUpdate 0.6s ease-out;
+}
+
+@keyframes renderFlashInitial {
+  0% {
+    border: 3px solid #3b82f6;
+    boxshadow: 0 0 0 0 rgba(59, 130, 246, 0.7);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)